frontend/public/components/modals/index.ts

```diff
@@ -54,3 +59,9 @@ export const clusterChannelModal = (props) => import('./cluster-channel-modal' /
 
 export const clusterUpdateModal = (props) => import('./cluster-update-modal' /* webpackChunkName: "cluster-update-modal" */)
   .then(m => m.clusterUpdateModal(props));
+
+export const taintModal = (props) => import('./taint-modal' /* webpackChunkName: "taint-modal" */)
```

Comment:
It should be `taintsModal` and `tolerationsModal` (plural) here and throughout. This is consistent with `annotationsModal` and `labelsModal`